### ISSUE TITLE
Mac OS Sierra `make check` failure

### ISSUE THREAD

**Issue by hughperkins** [vgough/encfs#329]
```
2/2 Test #2: scriptedtests ....................***Failed   20.32 sec
/Users/me/mygit/encfs/tests/normal.t.pl ... ok
Can't exec "attr": No such file or directory at /Users/me/mygit/encfs/tests/reverse.t.pl line 87.

#   Failed test 'symlink to '/' extended attributes can be read (return code was -1)'
#   at /Users/me/mygit/encfs/tests/reverse.t.pl line 89.
#          got: '-1'
#     expected: '0'
Can't exec "attr": No such file or directory at /Users/me/mygit/encfs/tests/reverse.t.pl line 87.

#   Failed test 'symlink to 'foo' extended attributes can be read (return code was -1)'
#   at /Users/me/mygit/encfs/tests/reverse.t.pl line 89.
#          got: '-1'
#     expected: '0'
Can't exec "attr": No such file or directory at /Users/me/mygit/encfs/tests/reverse.t.pl line 87.

#   Failed test 'symlink to '/1/2/3/4/5/6/7/8/9/10/11/12/13/14/15/15/17/18' extended attributes can be read (return code was -1)'
#   at /Users/me/mygit/encfs/tests/reverse.t.pl line 89.
#          got: '-1'
#     expected: '0'
Can't exec "attr": No such file or directory at /Users/me/mygit/encfs/tests/reverse.t.pl line 87.

#   Failed test 'symlink to '!§$%&/()\<>\#+=' extended attributes can be read (return code was -1)'
#   at /Users/me/mygit/encfs/tests/reverse.t.pl line 89.
#          got: '-1'
#     expected: '0'
Can't exec "attr": No such file or directory at /Users/me/mygit/encfs/tests/reverse.t.pl line 87.

#   Failed test 'symlink to '/var/folders/fn/asdasd/T//encfs-reverse-tests-bRQS/plain/foo' extended attributes can be read (return code was -1)'
#   at /Users/me/mygit/encfs/tests/reverse.t.pl line 89.
#          got: '-1'
#     expected: '0'
# Looks like you failed 5 tests of 25.
/Users/me/mygit/encfs/tests/reverse.t.pl .. 
Dubious, test returned 5 (wstat 1280, 0x500)
Failed 5/25 subtests 

```

**Comment by hughperkins** [comment]
this is on latest github `master`:
```
$ git log -n 3
commit 8cc6639f64ef8f3000a1323f164b17c8cc1c4860
Author: benrubson <ben.rubson@gmail.com>
Date:   Sun May 28 17:01:43 2017 +0200

    Make CircleCI run make check

commit ea67ef11f5a4be1f149168deecbaa1d222df0257
Author: Johannes Bornhold <johannes@bornhold.name>
Date:   Tue Apr 11 23:17:20 2017 +0200

    Add check for reading the configuration from a pipe

commit 59378ae7fe3d7293a92f44eaa2316a6d99d4ee1b
Author: Johannes Bornhold <johannes@bornhold.name>
Date:   Tue Apr 11 21:16:47 2017 +0200

    Read config into memory and pass its content to tinyxml2 (#253)
    
    tinyxml2 does not support to read from a named pipe, since it uses seek to
    figure out the file size. In the case of a pipe this is not possible.
    
    Reading the file content into memory and passing the content to tinyxml2 allows
    to mitigate this problem.
```

**Comment by benrubson** [comment]
Mac OS does not have the `attr` command.
Sounds like we should modify `reverse.pl` to use the `xattr` command with Mac OS.

**Comment by benrubson** [comment]
#330 should help, could you confirm ?
Thank you 👍

**Comment by benrubson** [comment]
@rfjakob merged #330 to correct this issue 👍 
@hughperkins you should then be able to close this :)

**Comment by hughperkins** [comment]
Thanks! :-)
